monkey/monkey_island/cc/services/attack/technique_reports/T1136.py

```diff
@@ -0,0 +1,36 @@
+from monkey_island.cc.services.attack.technique_reports import AttackTechnique
+from monkey_island.cc.services.reporting.report import ReportService
+from common.utils.attack_utils import ScanStatus
+from monkey_island.cc.models import Monkey
+
+__author__ = "VakarisZ"
+
+
+class T1136(AttackTechnique):
+    tech_id = "T1136"
+    unscanned_msg = "Monkey didn't try creating a new user."
+    scanned_msg = ""
+    used_msg = "Monkey created a new user."
+
+    @staticmethod
+    def get_report_data():
+        data = {'title': T1136.technique_title()}
+
+        scanned_nodes = ReportService.get_scanned()
+        status = ScanStatus.UNSCANNED.value
+        for node in scanned_nodes:
+            if node['pba_results'] != 'None':
+                for pba in node['pba_results']:
+                    if pba['name'] == 'Backdoor user':
```

Comment:
Would be extra nice if we also checked "communicate as new user" pba here. So the ideal scenario is that we have two checking functions. One function checks "communicate as new user" output and another checks "Backdoor user" output. If output doesn't exist: `status = UNSCANNED. value` if `"Error" in output`: `status = SCANNED.value`, else: `status = USED.value`. What you should do is make these PBA's fail on linux and windows manually and see if we can grep "error" and then presumed it failed.

Reply:
I think that's already being checked in `post_breach/pba.py` (lines 87 to 94). What I get in `node['pba_results']['result']` is of the form `[<output> (type: str), <didn't get error> (type: bool)]`.

Reply:
Made some changes in my latest commit; please check it. Is that similar to what you were expecting?